README.md

```diff
@@ -3,115 +3,94 @@
+
+	1. Download and install a Subversion client. [SilkSVN](https://sliksvn.com/download/) is recommended.
+	2. Download and install [gstreamer](https://gstreamer.freedesktop.org/download/). (When the installer prompts you, select the complete installation.)
+	3. Download the Psychtoolbox MATLAB [installer function](https://github.com/Psychtoolbox-3/Psychtoolbox-3/blob/master/Psychtoolbox/DownloadPsychtoolbox.m) from the PsychToolbox GitHub page.
+	4. Run the installer function in MATLAB with a target installation location as a string input argument, e.g. `DownloadPsychtoolbox('C:\')`, and follow the instructions that appear in the MATLAB command window.
 
-Below are a few minimum hardware requirements for both PCs.  These are more of a guide than a requirement as it depends on the type of experiments you wish to run. 
 
-**Processor:** Intel Core i3 7100, 3.9 GHz  
-**Graphics:** NVIDIA NVS 510 (for three screen support)  
-**Memory:** 16 GB 2133MHz DDR4 RAM, 1.2 V  
-**Storage:** 1TB HD, 7200rpm, 64MB Cache  
-**OS:** Windows 7 64-bit  
+Similar to the hardware requirements, software requirements for a SC will depend on the experiment: if acquiring data through a NI-DAQ, the SC will require the MATLAB Data Acquisition Toolbox and MATLAB [NI-DAQmx support package](https://uk.mathworks.com/hardware-support/nidaqmx.html) 
 
 ## Installation
 
-Before starting, ensure the above toolboxes and packages are installed.  PsychToobox can not be installed via the MATLAB AddOns browser.  See [Installing PsychToobox](#Installing-PsychToolbox) for install instructions.  
+Before starting, ensure you have read and followed the above [requirements section](#requirements).
 
-It is highly recommended to install Rigbox via the [Git Bash](https://git-scm.com/download/win) terminal*. 
+Here we provide brief instructions for installing Rigbox via Git. (If not already installed, download and install [Git](https://git-scm.com/download/win), and if unsure which options to select during installation, accept the installer defaults).
 
-1. To install Rigbox, run the following commands in the Git Bash terminal to clone the repository from GitHub to your local machine.
+1. Clone the repository from GitHub. In your git terminal, run:
 ```
 git clone --recurse-submodules https://github.com/cortex-lab/Rigbox
 ```
-2. Run the `addRigboxPaths.m` function in MATLAB (found in the Rigbox directory) then restart the program.  This adds all required folders and functions to your MATLAB path.  *Note*: Do __not__ manually add all Rigbox folders and subfolders to the paths!**
+(*Note*: It is **not** recommended to clone directly into the MATLAB folder).
 
-\* Accepting all installer defaults will suffice.  
-** To add the paths temporarily for testing:
+2. Add all required Rigbox folders and functions to your MATLAB path. In MATLAB, navigate to the Rigbox root directory (where Rigbox was cloned), and run:
```

Comment:
I like explaining commands before telling people to run commands. I like the code block because it differentiates it from the text, but don't feel strongly about this.